include/ebpf_result.h

```diff
@@ -54,41 +54,38 @@ extern "C"
         // The program is too large.
         EBPF_PROGRAM_TOO_LARGE,
 
-        // RPC exception.
+        // An RPC exception occurred.
         EBPF_RPC_EXCEPTION,
 
-        // Handle is already initialized.
+        // The handle was already initialized.
         EBPF_ALREADY_INITIALIZED,
 
         // Generic failure code for all other errors.
         EBPF_FAILED,
 
         // Operation is not supported.
-        EBPF_ERROR_NOT_SUPPORTED,
+        EBPF_OPERATION_NOT_SUPPORTED,
 
-        // The requested item was not found.
-        EBPF_ERROR_NOT_FOUND,
+        // The requested key was not found.
+        EBPF_KEY_NOT_FOUND,
 
         // Access was denied for the requested operation.
-        EBPF_ERROR_ACCESS_DENIED,
+        EBPF_ACCESS_DENIED,
 
         // The operation was blocked by policy.
-        EBPF_ERROR_BLOCKED_BY_POLICY,
+        EBPF_BLOCKED_BY_POLICY,
```

Comment:
We should add a comment when exactly blocked by policy should be used as opposed to access denied.

Reply:
example, capture as below change illustrates [hvci=> blocked by policy]